docs/setup.rst

```diff
@@ -289,6 +289,7 @@ writing. Windows is not and will never be supported.
     *   ``optipng`` for optimizing thumbnails
     *   ``gnupg`` for handling encrypted documents
     *   ``libpq-dev`` for PostgreSQL
+    *   ``mariadb-client`` for MariaDB
```

Comment:
Would `default-libmysqlclient-dev` be required for the initial install?

Reply:
Yes, I added that as well.